src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Mod_Float.txt

```diff
@@ -0,0 +1,43 @@
+#SETUP: NumberIsFloat
+
+>> Mod(1E+307, 3)
+1
+
+>> Mod(1E+307, -3)
+-2
+
+>> Mod(-1E+307, 3)
+2
+
+>> Mod(-1E+307, -3)
+-1
+
+>> Mod(1.7976931348623157E+300, 3)
+1
+
+>> Mod(1E+305, 123)
+112
+
+>> Mod(1.2345E+260, 67.89)
+41.28195726574958
+
+>> Mod(1.7976931348623157E+308, 25)
+18
+
+>> Mod(1E+13, 123)
+16
+
+>> Mod(1E+18, 123)
+16
+
+>> Mod(1E+25, 123)
+4
+
+>> Mod(1E+25, -123)
+-119
+
+>> Mod(10^200,-10^-200)
```

Comment:
>Mod(10^200,-10^-200) [](http://example.com/codeflow?start=3&length=20)

For some of those tests, if we want to leave them here, it may be worth calling out that they are what we show here because of floating point semantics.

Reply:
add comment.